src/java.base/share/classes/jdk/internal/module/ModuleHashes.java

```diff
@@ -116,17 +117,18 @@ private static byte[] computeHash(ModuleReader reader, String algorithm) {
         }
         try {
             byte[] buf = new byte[32*1024];
```

Comment:
Can you move the declaration of bug to before the try? That would reduce the nesting.